Eatery/Controllers/Eateries/EateriesViewController.swift

```diff
@@ -210,6 +211,25 @@ class EateriesViewController: UIViewController, MenuButtonsDelegate, CLLocationM
         view.addGestureRecognizer(collectionView.panGestureRecognizer)
     }
     
+    func presentGivingDayPopup(){
+        let todayDate = Date()
+        let givingDayStart : Date = {
+            let formatter = DateFormatter()
+            formatter.dateFormat = "MM/dd/yy HH:mm"
+            return formatter.date(from: "03/14/19 00:00")!
```

Comment:
this ! hurts me. Is there any way to unwrap this safely. If you can't parse the dates you can just return out of the function?

Reply:
We should add the timepiece pod; It'll be good here, and after backend finishes the refactor we'll need it anyways.

Reply:
> We should add the timepiece pod; It'll be good here, and after backend finishes the refactor we'll need it anyways.

I spoke with Young & Daniel and they suggested to always try and minimize the amount of dependencies that our apps have bc if the Timepiece pod goes down, Eatery goes down with it. Since I was really easily able to do this, I don't think it is crucial to use timepiece.

Reply:
We can use the Calendar APIs otherwise. 

https://developer.apple.com/documentation/foundation/nscalendar/1407609-date

Reply:
Or, pull out the date fomatter from the closures and parse the dates from the strings you have now, which will also let you guard for nil values.